packages/react-next/package.json

```diff
@@ -34,6 +34,7 @@
     "update-api": "just-scripts update-api"
   },
   "devDependencies": {
+    "@fluentui/common-styles": "^0.1.0",
```

Comment:
Moved to devDependencies, since OUFR's package.json lists office-ui-fabric-core under devDependencies. Just wanted to double-check that's correct @ecraig12345?

Reply:
I think so? Not the most familiar with scss build but I'm pretty sure these common scss files are only processed at build time.